common/src/main/java/com/viaversion/viaversion/protocols/v1_8to1_9/storage/EntityTracker1_9.java

```diff
@@ -185,7 +191,9 @@ public void handleEntityData(int entityId, List<EntityData> entityDataList) {
                     byte data = (byte) entityData.getValue();
                     if (entityId != getProvidedEntityId() && Via.getConfig().isShieldBlocking()) {
                         if ((data & 0x10) == 0x10) {
-                            if (validBlocking.contains(entityId)) {
+                            // If sword blocking is done through consumables, don't add a shield.
+                            if (validBlocking.contains(entityId) &&
+                                    user().getProtocolInfo().protocolVersion().olderThan(ProtocolVersion.v1_21_4)) {
```

Comment:
Move the protocol version check up to the ``isShieldBlocking`` if or is there a specific reason to put it here?

Reply:
I think that can work. Before I also had to check for `swordBlockingViaConsumable`, so it made sense to support shields that were present before a config reload